### PR TITLE
Rs builders big pr

### PR DESCRIPTION
**Pull request by julia-zack**
Create tests related to the new ErpRedeemScriptBuilders
Create tests for NonStandardErpRedeemScriptBuilderFactory
Move builders files to bitcoin package

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/NonStandardErpRedeemScriptBuilderFactoryTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        networkParameters = NetworkParameters.fromID(NetworkParameters.ID_TESTNET);
+
+        ErpRedeemScriptBuilder builder =
+            NonStandardErpRedeemScriptBuilderFactory.getNonStandardErpRedeemScriptBuilder(activations, networkParameters);
+
+        Assertions.assertTrue(builder instanceof NonStandardErpRedeemScriptBuilderHardcoded);
+    }
+
+    @Test
+    void preRSKIP284_doesnt_return_builderHardcoded_mainnet() {
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/NonStandardErpRedeemScriptBuilderFactoryTest.java:36]: Let's parametrize  the network in these tests

**Review by marcos-iov — COMMENTED** [review]
Well done! Looking really good. Just a small comment to parametrize some tests, and also check that some files don't have the extra empty line at the end

File: rskj-core/src/test/java/co/rsk/peg/NonStandardErpRedeemScriptBuilderFactoryTest.java (hunk 1 above)
Comment (julia-zack) [reply rskj-core/src/test/java/co/rsk/peg/NonStandardErpRedeemScriptBuilderFactoryTest.java:36]: but they expect different assertions! in testnet should be true and mainnet should be false.
I think I could parameterize some other tests tho

**Review by marcos-iov — APPROVED** [review]

File: rskj-core/src/main/java/co/rsk/peg/ErpFederationCreationException.java [hunk 2]
```diff
@@ -9,9 +9,7 @@ public class ErpFederationCreationException extends RuntimeException {
 
     public enum Reason {
         NULL_OR_EMPTY_EMERGENCY_KEYS,
-        INVALID_INTERNAL_REDEEM_SCRIPTS,
-        INVALID_CSV_VALUE,
-        HARDCODED_LEGACY_ERP_TESTNET_REDEEM_SCRIPT
+        REDEEM_SCRIPT_CREATION_FAILED
```
Comment (nathanieliov) [inline rskj-core/src/main/java/co/rsk/peg/ErpFederationCreationException.java:12]: .

**Pull request merged**